### ISSUE TITLE
Add comments route names

### ISSUE THREAD

**Issue by fey** [laravelista/comments#76]
I propose add to routes names.
For example:
```
<?php

Route::post('comments', config('comments.controller') . '@store')->name('comments.store');
Route::delete('comments/{comment}', config('comments.controller') . '@destroy')->name('comments.destroy');
Route::put('comments/{comment}', config('comments.controller') . '@update')->name('comments.update');
Route::post('comments/{comment}', config('comments.controller') . '@reply')->name('comments.reply');
```

**Comment by mabasic** [comment]
Is there a specific case why you think this would be beneficial?

There are only a few routes, I don't see why I would name them, but if you have a reason please tell.

**Comment by fey** [comment]
I have multiple forms that contains hardcoded routes
```
action="{{ url('comments/' . $comment->id) }}"
```
If we change routes for comments, we should edit all whole code.

**Comment by mabasic** [comment]
Seems logical 😊. I will accept this if you send a PR, otherwise you will have to wait until I circle back to working on open source.

**Comment by mabasic** [comment]
Included in version `3.3.0`. Thank you :heart:
